developers/blobstream.md

```diff
@@ -48,18 +46,37 @@ in the block range from the previous update to the current update, making
 accessible all Celestia data roots (verifiable with a Merkle inclusion proof
 against the stored Merkle root) to rollups.
 
-TODO: GRAPHIC HERE
-
 Blobstream X is built and deployed with
 [Succinct’s unified proving stack](https://succinct.xyz/).
 
-## Integrate with BlobstreamX
+![blobstream x draft diagram](/img/blobstream/blobstream_x_draft_diagram.png)
+
+## Integrate with Blobstream X
+
+The following docs go over how developers can integrate Blobstream X.
+
+### Overview
+
+You can [find the repository for Blobstream X](https://github.com/succinctlabs/blobstreamx)
+along with code for:
+
+- The Blobstream X smart contract - [`BlobstreamX.sol`](https://github.com/succinctlabs/blobstreamx/blob/main/contracts/src/BlobstreamX.sol)
+- [The Blobstream X circuits](https://alpha.succinct.xyz/celestia/blobstreamx)
```

Comment:
probably also add links to the golang binding : https://github.com/succinctlabs/blobstreamx/tree/main/bindings